### PR TITLE
Adding initial code and installing black, mypy, flake8, etc

### PR DESCRIPTION
**Pull request by cl3t0**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (brenobertone) [inline Makefile:1]: Where is the `make run`?

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "computer-vision"
+version = "0.1.0"
+description = ""
+authors = ["Pedro Cleto <cl3t0.sh@gmail.com>"]
```
Comment (brenobertone) [inline pyproject.toml]: Where is me? 👀

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[mypy]
+files = .
+ignore_missing_imports = True
+
+[flake8]
+max-complexity = 17
```
Comment (brenobertone) [inline setup.cfg]: Lower this to 10

**Review by brenobertone — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (cl3t0) [reply Makefile:1]: I will create it after the separation of camera logic and roboRIO communication.

File: pyproject.toml (hunk 2 above)
Comment (cl3t0) [reply pyproject.toml]: Added.

File: setup.cfg (hunk 3 above)
Comment (cl3t0) [reply setup.cfg]: I lowered it to 11.

**Review by brenobertone — APPROVED** [review]

**Pull request merged**